compile_flags.txt

```diff
@@ -86,6 +88,8 @@
 -DV8_TARGET_ARCH_X64
 -DV8_TARGET_OS_LINUX
 -DV8_TYPED_ARRAY_MAX_SIZE_IN_HEAP=64
+-DV8_USE_PERFETTO
+-DPERFETTO_ENABLE_LEGACY_TRACE_EVENTS=1
```

Comment:
I didn't seem to need to define this in the internal repo. It looks like V8's trace-categories.h file defines it.

Reply:
Yeah, it's picked up by v8's gn build that we use for the internal repo.